src/main/java/com/google/code/or/OpenReplicator.java

```diff
@@ -310,14 +313,24 @@ public Long millisSinceLastEvent() {
 		return binlogParser.millisSinceLastEvent();
 	}
 
+	public boolean isStopOnEOF() {
+		return stopOnEOF;
+	}
+
+	public void setStopOnEOF(boolean stopOnEOF) {

```

Comment:
raising might make a certain degree of sense.  There's a lot of parameters here that don't actually affect run-time behavior (ie, they really should be set only in the initializer).